Classes/Domain/Model/Product/Tea.php

```diff
@@ -54,7 +54,7 @@ public function setDescription(string $description): void
 
     public function getImage(): ?FileReference
     {
-        if ($this->image instanceof LazyLoadingProxy) {
+        if ($this->image instanceof LazyLoadingProxy) { // @phpstan-ignore-line
```

Comment:
The code sniffer needs this to be on the line above.

But apart from that, I'd prefer to have the issues in a baseline file (like [in this project](https://github.com/sabberworm/PHP-CSS-Parser)) so we have a list of things we probably need to fix.